src/components/ha-cards.html

```diff
@@ -143,6 +143,14 @@
       });
   };
 
+  const iterateDomain = (collection, func) => {
+    Object.keys(collection)
```

Comment:
Use Object.value(collection) instead - you won't need to map key->value then.

Reply:
`Object.values` needs to be polyfilled.

Reply:
It is already used in `panels/config/zwave/zwave-groups.html` (my bad)
Should we polyfill it in es5 only? In that case we need to "deprecate" to es5 Chrome 50-53 and FF 43-46

What do you say?

Reply:
I'm fine with that. Chrome and FF update very aggressively so I don't expect many users running old versions.

Reply:
So... should I revert it to use Object.keys again?

Reply:
Yeah, revert to Object.keys

Regardless I sent home-assistant/home-assistant#12091 to update min browser versions and I'll send a PR to add the polyfill